develop/app/Anser/Orchestrators/v2/CreateOrderOrchestrator.php

```diff
@@ -29,7 +29,8 @@ protected function definition()
 
         // $this->setCacheInstance($this->cache);
         // $this->setCacheOrchestratorKey('createOrder_1');
-        $this->setStep()->addAction("product_service", $this->productService->getAllProduct());
+        $this->setStep()->addAction("product_service", $this->productService->deleteProduct(22));
```

Comment:
Is this step really one of our actual orchestrator steps?
And we probably haven't started design the orchestrator yet.

Reply:
This is my test action to see if this action can be called correctly and this error has been fixed.